changelogs/unreleased/gh-8121-memtx-mvcc-replication-stream.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/replication
 
-* Fixed possibility of transaction conflict errors during replication stream
+* Fixed transaction conflict errors that could happen during replication stream
   (gh-8121).
```

Comment:
This sentence feels strange for me. Maybe `on applying a replication stream`?

Reply:
The changelog entry has been removed from this pull request (to base changelogs on 2.10), but it is still relevant for #8234.